registration/src/main/java/org/spine3/samples/lobby/registration/order/OrderAggregate.java

```diff
@@ -203,6 +197,27 @@ private void apply(OrderReservationCompleted event) {
     }
 
     @Apply
+    private void apply(OrderTotalsCalculated event) {
+        final Order.Builder state = getState().toBuilder();
+        state.setPrice(event.getTotal());
+        incrementState(state.build());
```

Comment:
Aggregates already do not need to call `incrementState()`. Can you migrate this code to latest state of `core-java`?